src/helpers/fixtures.js

```diff
@@ -60,7 +60,13 @@ let fixtureHelper = module.exports = {
         }
       }
 
-      if (options.query && !_.isEqual(options.query, fixture.request.query)) {
+      const filteredFixturedQuery = xhrHelper.getQueryStringObject({
+        url: 'http://localhost/?' + Object.keys(fixture.request.query || {}).map(function(key) {
```

Comment:
Is the fact you've hardcoded localhost here significant at all? I'm thinking not since the function actually only cares about the query, and not the rest of the provided URL?